packages/docusaurus-plugin-vercel-analytics/src/options.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import {Joi} from '@docusaurus/utils-validation';
+import type {OptionValidationContext} from '@docusaurus/types';
+
+export type PluginOptions = {
+  id: string;
+  mode: 'auto' | 'production' | 'development';
+  debug: boolean;
+};
+
+export type Options = Partial<PluginOptions>;
+
+export const DEFAULT_OPTIONS: Partial<PluginOptions> = {
+  mode: 'production',
```

Comment:
I think `undefined` ("auto") makes more sense as a default

For example `docusaurus build --dev` builds the site with node env "development" and we probably want it to impact the vercel plugin env in that case.

See also:
- https://docusaurus.io/docs/next/cli#docusaurus-build-sitedir
- https://vercel.com/docs/analytics/package#mode

Reply:
I don't understand what you mean with `undefined` ("auto") but I changed the default value to auto which makes more sense